token/token.go

```diff
@@ -83,7 +82,9 @@ func NewManager(config ManagerConfiguration, options ...httpsupport.HTTPClientOp
 	}
 	tm.config = config
 
-	keysEndpoint := fmt.Sprintf("%s%s", config.GetAuthServiceURL(), config.GetAuthKeysPath())
+	authURL := httpsupport.AddTrailingSlashToURL(config.GetAuthServiceURL())
+	// TODO when we have a separate repo for Auth client we should use it to get the key path instead of hardcoding "api/token/keys"
+	keysEndpoint := fmt.Sprintf("%s%s", authURL, "api/token/keys")
```

Comment:
it should be `api/token/keys` OR `/api/token/keys` ??  see `/`.

Reply:
It should be `api/token/keys`. We are adding trailing slash to auth url (see line 85) if it doesn't already have. So, the authURL will have `/` in the end.